docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.7'
```

Comment:
I think if we're saying in the general docs that these docker compose template files can be used for local deployment, then we should set them up to work with minio, otherwise we'll have to give instructions for how to set up an s3 bucket.

Reply:
Actually ignore this comment since as you point out we'll probably just update the deploy to use local file storage by default